src/celeritas/ext/detail/GeantPhysicsList.cc

```diff
@@ -48,11 +48,11 @@ namespace detail
  */
 GeantPhysicsList::GeantPhysicsList(Options const& options) : options_(options)
 {
-    // Set EM options
+    // Set EM options using limits from G4EmParameters
     auto& em_parameters = *G4EmParameters::Instance();
-    CELER_VALIDATE(options_.em_bins_per_decade > 0,
+    CELER_VALIDATE(options_.em_bins_per_decade >= 5,
                    << "number of EM bins per decade="
-                   << options.em_bins_per_decade << " (must be positive)");
+                   << options.em_bins_per_decade << " (must be at least 5)");
```

Comment:
I initially thought this was because `GeantModelImporter`, but looking again, it divides the number of bins per decade by `6`. So now I'm confused, why `5`?

Reply:
In G4EmParameters, it will *ignore* the value given if less than 5 or over a million. I think an assertion is more appropriate than a warning.

Reply:
Ohh okay. So we end up with at least 5 bins for macroscopic process XS tables and at least 3 for the models.